daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -31,7 +31,6 @@
 // * 6 -- 2019-05-27: Add enum type.
 //        2019-06-04: Add BuiltinFunction.{TEXT_FROM_CODE_POINTS, TEXT_TO_CODE_POINTS}
 //        2019-06-12: Add Package.interned_package_ids and PackageRef.interned_id
-//        2019-07-04: Transaction submitters must be in contract key maintainers when looking up. See #1866.
```

Comment:
Are we pretending this never happened?

Reply:
I don’t think it makes sense to keep a history for `1.dev`.

Reply:
The comment was wrong, the check was performed in `1.dev` only.

Reply:
I see. Thanks for the explanation.